packages/webapp/src/tests/util.test.js

```diff
@@ -13,19 +13,25 @@
  *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
  */
 
+import { expect, describe, test } from 'vitest';
```

Comment:
This is totally unrelated to this PR but since I added automatically running tests in the CI I had to fix this one so it wouldn't fail.